Src/Common/Domain/implementation/VolunteerOpportunity.cs

```diff
@@ -8,42 +9,60 @@ public class VolunteerOpportunity : IVolunteerOpportunity
         #region Constructors
 
         public VolunteerOpportunity()
-            : this(null)
+            : this(null, null)
         {
         }
 
-        public VolunteerOpportunity(IUser createdBy)
+        public VolunteerOpportunity(IUser createdBy, IOrganization organization)
         {
             CreatedBy = createdBy;
+            Organization = organization;
 
             RecordId = 0;
-            StartTime = DateTime.Today;
-            StopTime = DateTime.Today;
+
+            ProjectName = string.Empty;

```

Comment:
If a project is going to be a separate entity altogether, I would make the most sense to just store the ProjectRecordId here as a foreign key to the Project. Then you can just grab those two pieces of information from the Project itself. That allows the project to be renamed without having to go update the Volunteer Opportunities.